flake8diff/vcs/git.py

```diff
@@ -65,3 +69,7 @@ def changed_files(self):
         return filter(self.filter_file,
                       iter(_execute(' '.join(command))
                            .splitlines()))
+
+    def _is_git_repository(self):
+        return _execute(
+            '{vcs} status'.format(vcs=self.vcs), strict=True, log_errors=False)
```

Comment:
Yeah, I think there can be 1 more corner case: bare repositories. And both my and your checks will fail :) I'll skip it for now as well.